elements/gnome-sdk-sdk/WebKitGTK-endless-arm.bst

```diff
@@ -0,0 +1,94 @@
+# Replaces sdk/WebKitGTK.bst from gnome-sdk.
+# <https://gitlab.gnome.org/GNOME/gnome-build-meta/blob/gnome-3-36/elements/sdk/WebKitGTK.bst>
+
+# Because the upstream version of this is required by other elements in
+# gnome-sdk, we need to keep the existing element intact, and put this one on
+# top of it during integration by using an overlap whitelist.
+
+kind: cmake
+
+sources:
+- kind: tar
+  url: webkitgtk_org:webkitgtk-2.27.91.tar.xz
+  ref: 30411781f620c33f5542a745930b48a19a6072bea0e68982c1ad863d1148bdea
+- kind: patch
+  path: gnome-sdk-files/webkitgtk/gtk-doc-introspection-cross-compiling.patch
+- kind: local
+  path: gnome-sdk-files/webkitgtk/toolchain.i686
+- kind: local
+  path: gnome-sdk-files/webkitgtk/toolchain.arm
+
+build-depends:
+- freedesktop-sdk.bst:components/gperf.bst
+- freedesktop-sdk.bst:components/perl.bst
+- freedesktop-sdk.bst:components/ruby.bst
+- freedesktop-sdk.bst:public-stacks/buildsystem-cmake.bst
+
+depends:
+- freedesktop-sdk.bst:bootstrap-import.bst
+- freedesktop-sdk.bst:components/hyphen.bst
+- freedesktop-sdk.bst:components/libtasn1.bst
+- freedesktop-sdk.bst:components/libwebp.bst
+- freedesktop-sdk.bst:components/libxslt.bst
+- freedesktop-sdk.bst:components/wayland.bst
+- freedesktop-sdk.bst:components/xorg-lib-xt.bst
+- gnome-sdk-sdk/gtk+-3.bst
+- gnome-sdk.bst:sdk/at-spi2-core.bst
+- gnome-sdk.bst:sdk/brotli.bst
+- gnome-sdk.bst:sdk/enchant-2.bst
+- gnome-sdk.bst:sdk/geoclue.bst
+- gnome-sdk.bst:sdk/gobject-introspection.bst
+- gnome-sdk.bst:sdk/gst-plugins-base.bst
+- gnome-sdk.bst:sdk/libnotify.bst
+- gnome-sdk.bst:sdk/libsecret.bst
+- gnome-sdk.bst:sdk/libsoup.bst
+- gnome-sdk.bst:sdk/libwpe.bst
+- gnome-sdk.bst:sdk/openjpeg.bst
+- gnome-sdk.bst:sdk/pango.bst
+- gnome-sdk.bst:sdk/woff2.bst
+- gnome-sdk.bst:sdk/wpebackend-fdo.bst
+
+runtime-depends:
+- gnome-sdk-sdk/gst-plugins-good.bst
+- gnome-sdk.bst:sdk/gst-libav.bst
+- gnome-sdk.bst:sdk/gst-plugins-bad.bst
+
+# dummy dependency so this is staged on top of the element from gnome-sdk
+- gnome-sdk.bst:sdk/WebKitGTK.bst
+
+variables:
+  webkitgtk_toolchain: ''
+  endless_arch_options: ''
+  (?):
+  - arch == "i686" or arch == "arm":
+      webkitgtk_toolchain: -DCMAKE_BUILD_TYPE=Release -DCMAKE_TOOLCHAIN_FILE=toolchain.%{arch}
+  - arch == "arm":
+      endless_arch_options: |
+        -DENABLE_GLES2=ON
+  cmake-local: >-
+    -DPORT=GTK
+    -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+    %{webkitgtk_toolchain}
+    %{endless_arch_options}
+(?):
+- arch == "i686":
```

Comment:
I don't think we build for i686, but perhaps this is inherited from the upstream element?

Reply:
Indeed, that one's from upstream :)